src/libraries/System.Private.Uri/tests/FunctionalTests/IriTest.cs

```diff
@@ -110,7 +110,8 @@ public void Iri_804110_TryCreateUri_ShouldNotThrowIndexOutOfRange()
 
             Assert.Equal(
                 "http://con.tosoco.ntosoc.com/abcdefghi/jk" + "%C8%F3%B7%A2%B7%BF%B2%FA",
-                resultUri.ToString());
+                resultUri.ToString(),
+                StringComparer.OrdinalIgnoreCase);
```

Comment:
This is an example of where the output would be different - the casing from the input would be preserved instead of upper-casing the hex chars